bpaotu/bpaotu/models.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+import logging
+logger = logging.getLogger("rainbow")
+
+
+class ImportLog(models.Model):
```

Comment:
can we rename this to ImportOntologyLog? Then if we have any other classes of bug we display, we can add them :-)

Reply:
Fixed.